README.md

```diff
@@ -135,6 +135,7 @@ Currently **officially** using Airflow:
 1. [Betterment](https://www.betterment.com/) [[@betterment](https://github.com/Betterment)]
 1. [BlaBlaCar](https://www.blablacar.com) [[@puckel](https://github.com/puckel) & [@wmorin](https://github.com/wmorin)]
 1. [Bloc](https://www.bloc.io) [[@dpaola2](https://github.com/dpaola2)]
+1. [Bloomberg](https://www.techatbloomberg.com) [[@bloomberg](https://github.com/bloomberg)]
```

Comment:
should be something like ``[Bloomberg](https://www.techatbloomberg.com) [[@kpfleming](kpfleming)]``

Reply:
Why is that? There are plenty of other examples in this document which refer to company GitHub organizations, like Betterment on line 135.

Reply:
I am not the one ship those changes(e.g betterment). If you look at other examples, most of the examples list with who is/are the point of contact from the company.

Reply:
That's fine; if we need to list someone who is our point of contact for Airflow usage, we can do that. @dimberman are you OK with being listed here as our point of contact?

Reply:
thanks @kpfleming @dimberman